_episodes/14-supplemental-vscode.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Supplemental: Using Git from VS-Code"
+teaching: 10
+exercises: 0
+questions:
+- "How can I use Git with VS-Code?"
+objectives:
+- "Understand how to use Git from VS-Code."
+keypoints:
+- "Using VS-Code's Git integration allows you to version control a project over time."
+---
+
+
+Version control can be very useful when developing data analysis scripts. For
+that reason, the popular development environment
+[Visual Studio Code ][https://code.visualstudio.com/] for the R programming language has built-in
+integration with Git. While some advanced Git features still require the
+command-line, VS-Code has a nice interface for many common Git operations.
```

Comment:
- We could start by defining the term IDE and providing some points on their advantages.
- Perhaps mention that other IDEs have Git integration (maybe give a name or two), but we are focussed on VS-Code because it is popular, cross-platform, and freely available. 
- It would be helpful to provide a link to VS-Code: https://code.visualstudio.com/ or platform-specific installation instructions (I went here for Linux https://code.visualstudio.com/docs/setup/linux).
- The section on R came with the original carpentries workshop - but has never been taught in the internship - so I don't think we need to mention R specifically. From experience, python is the most common language, followed by R, BASH and some C++/Fortran.